apps/condo/domains/organization/schema/RegisterNewOrganizationService.js

```diff
@@ -17,11 +17,10 @@ const RegisterNewOrganizationService = new GQLCustomSchema('RegisterNewOrganizat
                 if (!context.authedItem.id) throw new Error('[error] User is not authenticated')
                 const { data } = args
                 const extraData = { dv: data.dv, sender: data.sender }
-
                 const organization = await createOrganization(context, data)
-                const role = await createAdminRole(context, organization, extraData)
-                await createConfirmedEmployee(context, organization, context.authedItem, role, extraData)
-
+                const defaultRoles = await createDefaultRoles(context, organization, extraData)
```

Comment:
It’s looks like magic const 0. Are you sure that the first role is admin :)

Reply:
fixed by constructing object with keys as role names